src/array_api_typing/_namespace.py

```diff
@@ -0,0 +1,27 @@
+"""Static typing support for the array API standard."""
```

Comment:
It's a private module, so ruff wouldn't complain if we removed this.
```suggestion
```

We could also keep it and change it into something that's more specific to this module.